deployment/postgres-deployment.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: postgres-deployment
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: postgres
+  template:
+    metadata:
+      labels:
+        app: postgres
+    spec:
+      containers:
+      - name: postgres
+        image: latifa212/plantgame-postgres:latest
```

Comment:
the latest image from docker hub is not the same as the dockerfile in the database-initiation repo
maybe pushing it once again and using version numbers like this:
docker push latifa212/plantgame-postgres:0.0.1
this way we can use images which are perfect and work on newer ones at the same time

Reply:
the line should be like this after you pushed the version:
image: latifa212/plantgame-postgres:0.0.1